### PR TITLE
Update upgrade messaging

### PR DESCRIPTION
**Pull request by mooreds**
Make it more clear what upgrade options are and how FusionAuth cloud handles them.

### CODE REVIEW

File: site/docs/v1/tech/installation-guide/cloud.adoc [hunk 1]
```diff
@@ -203,46 +203,87 @@ image::installation-guides/cloud/deployments-manage.png[Manage deployment,width=
 === Upgrading a Deployment
 
 If your deployment is not running the latest version of FusionAuth, you may upgrade it.
+It is a good idea to run the latest released version of FusionAuth, which has the latest bug fixes, security updates and features.
```
Comment (jobannon) [inline site/docs/v1/tech/installation-guide/cloud.adoc:206]: > which has the latest bug fixes, security updates, and features.

adds comma

File: site/docs/v1/tech/installation-guide/cloud.adoc [hunk 2]
```diff
@@ -203,46 +203,87 @@ image::installation-guides/cloud/deployments-manage.png[Manage deployment,width=
...
+
+However, you will not be forced to upgrade on a certain schedule.
```
Comment (jobannon) [inline site/docs/v1/tech/installation-guide/cloud.adoc:208]: However, upgrades are optional and user-driven.

File: site/docs/v1/tech/installation-guide/cloud.adoc [hunk 3]
```diff
@@ -203,46 +203,87 @@ image::installation-guides/cloud/deployments-manage.png[Manage deployment,width=
...
 ====
 
-Perform an upgrade at a time that works for your users, your team and your applications by https://account.fusionauth.io/[logging into your account]. Record the time you began the upgrade.
+==== Upgrade Process
 
-Navigate to the [breadcrumb]#Deployments# tab. Manage the deployment, then choose the "Upgrade" option. Select the version you are upgrading to from the [field]#Version# dropdown.
+To upgrade a deployment from version `1.x-1` to version `1.x`:
 
-Confirm the upgrade:
+* Test `1.x` on your staging or development environment.
```
Comment (jobannon) [inline site/docs/v1/tech/installation-guide/cloud.adoc:261]: `your` -> `a`

File: site/docs/v1/tech/installation-guide/cloud.adoc [hunk 4]
```diff
@@ -203,46 +203,87 @@ image::installation-guides/cloud/deployments-manage.png[Manage deployment,width=
...
+* Select a time that work for your users, your team and your applications.
```
Comment (jobannon) [inline site/docs/v1/tech/installation-guide/cloud.adoc:262]: Select a time that works for your users, your team, and your applications.

File: site/docs/v1/tech/installation-guide/cloud.adoc [hunk 5]
```diff
@@ -203,46 +203,87 @@ image::installation-guides/cloud/deployments-manage.png[Manage deployment,width=
...
+* Record the time you began the upgrade.
+* Navigate to the [breadcrumb]#Deployments# tab. Manage the deployment, then choose the "Upgrade" option. Select `1.x` from the [field]#Version# dropdown.
+* Confirm you want to start the upgrade:
 
 image::installation-guides/cloud/deployments-upgrade-confirm.png[Upgrading a deployment,width=1200,role=bottom-cropped]
 
-After confirmation, the deployment will be in an "Upgrading" state until finished. You can monitor the upgrade by viewing the [breadcrumb]#Deployments# tab. If you need to programmatically monitor the upgrade, you can call the link:/docs/v1/tech/apis/system#retrieve-system-status[System Status API]; when it returns success, the deployment upgrade is complete.
+* After confirmation, the deployment will be in an "Upgrading" state until finished. Monitor the upgrade by viewing the [breadcrumb]#Deployments# tab. To programmatically monitor the upgrade, call the link:/docs/v1/tech/apis/system#retrieve-system-status[System Status API].
+* When the deployment is in the `Active` state in the user interface, or when the System Status API returns success, the deployment is upgraded to `1.x`.
```
Comment (jobannon) [inline site/docs/v1/tech/installation-guide/cloud.adoc:271]: > When the deployment is in the `Active` state in the user interface, or when the System Status API returns success, the deployment is upgraded to `1.x`.

An upgrade is complete when the user interface deployment information moves from "Upgrading" to "Active", or when the System Status API returns success.

File: site/docs/v1/tech/installation-guide/cloud.adoc [hunk 6]
```diff
@@ -203,46 +203,87 @@ image::installation-guides/cloud/deployments-manage.png[Manage deployment,width=
...
 You cannot downgrade a FusionAuth Cloud deployment version.
 
-==== Upgrades with a High-Availability Plan
+Please test upgrades before performing them on your production system.
 
-Upgrades with this tier have less downtime.
+==== Rolling Back From a Problematic Upgrade
 
-If the release notes indicate a database migration, the upgrade process will result in a few minutes of outage. Otherwise there will be no downtime.
+If an upgrade is problematic for whatever reason, and you identify the issues within your database backup retention period, open a https://account.fusionauth.io/account/support/[support ticket].
```
Comment (jobannon) [inline site/docs/v1/tech/installation-guide/cloud.adoc:281]: If an upgrade is problematic, and the offending issue is identified in the same timeframe as your database backup retention period, open a https://account.fusionauth.io/account/support/[support ticket].

Comment (jobannon) [reply site/docs/v1/tech/installation-guide/cloud.adoc:281]: Maybe 

> Please note, support tickets will be addressed per your Support Edition SLA, with deference granted towards the urgency of the situation.

File: site/docs/v1/tech/installation-guide/cloud.adoc [hunk 7]
```diff
@@ -203,46 +203,87 @@ image::installation-guides/cloud/deployments-manage.png[Manage deployment,width=
...
+|5 to 60 minutes
+
+|Business Cloud
+|5 to 60 minutes
+|5 to 60 minutes
 
-Due to the downtime, it is recommended that you schedule the upgrade for a low traffic period. Test the upgrade process on development or test servers first.
+|HA Cloud
+|5 to 60 minutes
+|seconds
```
Comment (jobannon) [inline site/docs/v1/tech/installation-guide/cloud.adoc:235]: > Short-lived duration (less than 60 seconds)

File: site/docs/v1/tech/installation-guide/_downtime-upgrade-limitation.adoc [hunk 8]
```diff
@@ -1,10 +1,24 @@
...
 
-The recommendation is that you automate the process and minimize downtime. For our FusionAuth cloud hosted solutions we use a configuration management tool and see downtime on the order of seconds.
+* Take down all nodes running `1.x-1`.
+* Upgrade the database schema by running the migrations, bringing the database to version `1.x`.
+* Upgrade the FusionAuth application and (if required) Elasticsearch packages, bringing your nodes to `1.x`.
+* Start all FusionAuth nodes.
 
-There's an https://github.com/FusionAuth/fusionauth-issues/issues/1240[open issue for zero downtime upgrades]. Please vote that up if this is important to you.
+The recommendation is that you automate the process and minimize downtime.
+For FusionAuth Cloud we use a configuration management tool and see downtime on the order of seconds.
```
Comment (jobannon) [inline site/docs/v1/tech/installation-guide/_downtime-upgrade-limitation.adoc:12]: > t tool and see downtime on the order of seconds.

For ha 🙁

**Review by jobannon — APPROVED** [review]
Thanks for writing dan!

File: site/docs/v1/tech/installation-guide/cloud.adoc (hunk 7 above)
Comment (mooreds) [reply site/docs/v1/tech/installation-guide/cloud.adoc:235]: That's not what I heard Brian say :)

Comment (mooreds) [reply site/docs/v1/tech/installation-guide/cloud.adoc]: Confirmed with Brian that it is seconds of downtime for user logins

**Pull request merged**